### PR TITLE
docs: add screenshots to webhook docs.

### PR DESCRIPTION
**Pull request by sashaostrikov**
Test plan:
Visual checks on local sg docs page.

Closes https://github.com/sourcegraph/sourcegraph/issues/45655

Screenshot files added [here](https://console.cloud.google.com/storage/browser/sourcegraph-assets/docs/images/administration/config/webhooks?pageState=(%22StorageObjectListTable%22:(%22f%22:%22%255B%255D%22))&folder=true&organizationId=true_&project=sourcegraph-de&prefix=&forceOnObjectsSortingFiltering=false)

### CODE REVIEW

**Comment by ryanslade** [comment]
I noticed that screenshots in other parts of our docs are at a different URL, for example:
https://sourcegraphstatic.com/docs/images/batch_changes/user-token-input.png

**Comment by sashaostrikov** [comment]
> I noticed that screenshots in other parts of our docs are at a different URL, for example: https://sourcegraphstatic.com/docs/images/batch_changes/user-token-input.png

@ryanslade Yep, I don't think that we have a strong convention here, I only found info about `storage.googleapis.com` on Handbook.

We currently have 145 results for [storage.googleapis.com](https://sourcegraph.sourcegraph.com/search?q=context:global+repo:%5Egithub%5C.com/sourcegraph/sourcegraph%24+storage.googleapis.com/sourcegraph-assets/&patternType=standard&sm=1) and 71 for [sourcegraphstatic](https://sourcegraph.sourcegraph.com/search?q=context:global+repo:%5Egithub%5C.com/sourcegraph/sourcegraph%24+sourcegraphstatic.com/docs/&patternType=standard&sm=1), I just went for a more popular option :)

File: doc/admin/config/webhooks.md [hunk 1]
```diff
@@ -25,16 +25,19 @@ Before adding an incoming webhook you should ensure that you have at least one [
 The incoming webhook will be configured to accept events from a specific code host connection based on its type and URN.
 
 1. Navigate to **Site Admin > Incoming webhooks**
-1. Click **Add webhook**
-1. Fill out the form:
+   ![Incoming webhooks page](https://storage.googleapis.com/sourcegraph-assets/docs/images/administration/config/webhooks/adding-webhook.png)
```
Comment (ryanslade) [inline doc/admin/config/webhooks.md:28]: The first two screenshots are the same, is that intentional?

Comment (sashaostrikov) [reply doc/admin/config/webhooks.md]: whoops, I've messed it up, thanks for finding this! fixed

Comment (ryanslade) [reply doc/admin/config/webhooks.md]: I'm getting an error loading the first screenshot, this one:
https://storage.googleapis.com/sourcegraph-assets/docs/images/administration/config/webhooks/incoming-webhook-page.png

Comment (ryanslade) [reply doc/admin/config/webhooks.md]: Screenshot 2022-12-16 at 12 31 48

Comment (sashaostrikov) [reply doc/admin/config/webhooks.md]: typo number 2, aha, fix incoming

**Review by ryanslade — APPROVED** [review]
Nice!

**Comment by mrnugget** [comment]
I don't know if you did it or not, but just in case: I usually run all images I put in the docs through [`ImageOptim.app`](https://imageoptim.com/mac) first to reduce file size.

**Comment by sashaostrikov** [comment]
@mrnugget nope, I'll do it and add this info to docs, because I haven't found this hint

**Pull request merged**